recipes/bel-alex73/choose_speaker_jhlfrfufyfn.ipynb

```diff
@@ -0,0 +1,343 @@
+{
```

Comment:
Can you rename this file? It looks a bit funky :)

Reply:
Fixed